bpfd-api/src/util.rs

```diff
@@ -8,15 +8,23 @@ pub mod directories {
     // is not being run as a service.
     //
     // ConfigurationDirectory: /etc/bpfd/
-    pub const CFGDIR:                 &str = "/etc/bpfd";
-    pub const CFGDIR_BPFD_CERTS:      &str = "/etc/bpfd/certs/bpfd";
-    pub const CFGDIR_CA_CERTS:        &str = "/etc/bpfd/certs/ca";
-    pub const CFGDIR_STATIC_PROGRAMS: &str = "/etc/bpfd/programs.d";
-    pub const CFGPATH_BPFD_CONFIG:    &str = "/etc/bpfd/bpfd.toml";
-    pub const CFGPATH_CA_CERTS_PEM:   &str = "/etc/bpfd/certs/ca/ca.pem";
-    pub const CFGPATH_CA_CERTS_KEY:   &str = "/etc/bpfd/certs/ca/ca.key";
-    pub const CFGPATH_BPFD_CERTS_PEM: &str = "/etc/bpfd/certs/bpfd/bpfd.pem";
-    pub const CFGPATH_BPFD_CERTS_KEY: &str = "/etc/bpfd/certs/bpfd/bpfd.key";
+    pub const CFGDIR:                       &str = "/etc/bpfd";
+    pub const CFGDIR_BPFCTL_CERTS:          &str = "/etc/bpfd/certs/bpfctl";
+    pub const CFGDIR_BPFD_CERTS:            &str = "/etc/bpfd/certs/bpfd";
+    pub const CFGDIR_BPFD_AGENT_CERTS:      &str = "/etc/bpfd/certs/bpfd-agent";
```

Comment:
Again Can't we just do `bpfd-client` and then those can be used by bpfd-agent, bpfctl or both? 

(I'm not sure just asking for the sake of review :) )

Reply:
I can if that is the way we want to go. Before I started there was a bpfctl and a gocounter cert, so was just following that model. So one cert for all clients?

Reply:
If that would work it actually would simplify all of this pretty dramatically... Like I don't want to have to add functionality to bpfd every time we add some sort of new client. 

And at the end of the day in any real production scenario I can't imagine BPFD would be managing it's own certs

Reply:
Now we have a ca, bpfd and bpfd-client cert created by bpfd if they don't exist. All clients (bpfctl, bpfd-agent, gocounter) share the single bpfd-client certs.